torch_encoder.py

```diff
@@ -23,13 +23,13 @@ class ImageTorchEncoder(Executor):
 
     Internally, :class:`ImageTorchEncoder` wraps the models from
     `torchvision.models`.
-    https://pytorch.org/docs/stable/torchvision/models.html
+    https://pytorch.org/vision/stable/models.html
 
     :param model_name: the name of the model. Supported models include
         ``alexnet``, `squeezenet1_0``,  ``vgg16``,
```

Comment:
I think the list is much larger, because all other resnets should be available as well

Reply:
Then we can remove it and just reference the link

Reply:
I'm now specifying that the list is not complete and that they should check the link.